splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceSystemProcedures.java

```diff
@@ -1177,15 +1177,25 @@ public SpliceSystemProcedures(DataDictionary dictionary) {
                             .build());
 
                     Procedure purgeDeletedRows = Procedure.newBuilder().name("SET_PURGE_DELETED_ROWS")
-                            .varchar("schemaName", 128)
-                            .varchar("tableName", 128)
+                            .catalog("schemaName")
+                            .catalog("tableName")
                             .varchar("enable", 5)
                             .numOutputParams(0)
                             .numResultSets(0)
                             .ownerClass(SpliceAdmin.class.getCanonicalName())
                             .build();
                     procedures.add(purgeDeletedRows);
 
+                    Procedure minRetentionPeriod = Procedure.newBuilder().name("SET_MIN_RETENTION_PERIOD")
+                            .varchar("schemaName", 128)
```

Comment:
You replaced varchar with catalog in SET_PURGE_DELETED_ROWS, but not here.

Reply:
It is not a big deal since `catalog` is aliasing `varchar(128)` type. I will fix it anyway.